proposals/devcontainer-features-distribution.md

```diff
@@ -0,0 +1,131 @@
+# Community Contribution and Discovery of Dev Container Features
+
+This specification defines a pattern where community members and organizations can author and self-publish [dev container 'features'](./devcontainer-features.md). 
+
+Goals include:
+
+- For community authors, a "self-served" mechanism for dev container feature publishing.
+- For community members, a largely self-served mechanism for marking features as "discoverable" by [supporting tools](../docs/specs/supporting-tools.md).
+- For users, ease of discoverability of features via [supporting tools](../docs/specs/supporting-tools.md).
```

Comment:
Curious, how would they discover features with the `supporting tools` ? 
I thought the users can consume these features with the `supporting tools`

Reply:
I will rephrase